gyp.js

```diff
@@ -30,7 +30,7 @@ function runGyp (opts, cb) {
   function runStep () {
     var command = gyp.todo.shift()
     if (!command) {
-      return cb()
+      return cb ? cb() : undefined
```

Comment:
Is `cb` ever undefined? I ask as this function later relies on it... (1/2)

Reply:
In test `gyp is invoked with correct arguments, debug mode`. I added a callback to the test instead.